entities/auctioneer.py

```diff
@@ -15,6 +16,9 @@ def __init__(self, env, region):
         self.auctionable_shipments = []
```

Comment:
already fixed the unlist functions, replaced the for loops (which was not working for the container bids), with list comprehensions.